README.md

```diff
@@ -135,8 +139,17 @@ Note that versions of Node.JS v8 prior to `v8.12.0` will NOT work, because OpenT
 > Client instrumentation for the browser is **experimental** and mostly **unspecified**. If you are interested in
 > helping out, get in touch with the [Client Instrumentation SIG][client-instrumentation-sig].
 
-There is currently no list of officially supported browsers. OpenTelemetry is developed using standard web
-technologies and aims to work in currently supported versions of major browsers.
+Rather than define versions of specific browsers / runtimes, OpenTelemetry sets the minimum supported version based on the
+underlying Language features used.
```

Comment:
```suggestion
underlying language features used.
```

or if you want to keep this as upper case, you should change the paragraph below to also use upper case

Reply:
fixed in https://github.com/open-telemetry/opentelemetry-js/pull/5284/commits/2610521b1638265dd5f12bcdd72e5a36d19157c7 🙂